Source/WebKit/UIProcess/API/Cocoa/WKWebViewInternal.h

```diff
@@ -118,7 +118,7 @@ class ViewGestureController;
 
 @class WKContentView;
 @class WKPasswordView;
-@class WKSafeBrowsingWarning;
+@class WKWarningView;
```

Comment:
Could we name this _WKWarningView because it's not intended to be used by third parties?